### PR TITLE
Add Jenkins

### PR DESCRIPTION
**Pull request by ManthanKeim**
Jenkins contribution in Hacktoberfest 2019.

### CODE REVIEW

**Comment by crweiner** [comment]
Hi @ManthanKeim !

Is Jenkins offering their own unique swag for Hacktoberfest this year or are they encouraging people to contribute to their repos to earn the official DO/Dev swag?

If they are offering their own unique swag, then I'd be happy to add them here. If so, they would need to be added to the Least Involvement to Most Involvement section of the list as well.

**Comment by ManthanKeim** [comment]
> Hi @ManthanKeim !
> 
> Is Jenkins offering their own unique swag for Hacktoberfest this year or are they encouraging people to contribute to their repos to earn the official DO/Dev swag?
> 
> If they are offering their own unique swag, then I'd be happy to add them here. If so, they would need to be added to the Least Involvement to Most Involvement section of the list as well.

 there will be swag for few dozens of top contributors for sure, but more information to be defined next week.

File: README.md [hunk 1]
```diff
@@ -149,6 +149,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [jabref](https://github.com/search?&q=user%3Ajabref+type:issue&state=open&type=Issues)
 - **Notes**: More details in [the blog post](https://www.jabref.org/hacktoberfest/2019.html).
 
+#### **Jenkins**
+
+- **Swag**: limited edition swag
+- **Requirements**: PR to any repositories under [any repo of Jenkins](https://issues.jenkins-ci.org/browse/JENKINS-59817?filter=-4). The PR must be at least raised in October 2019 and you should not be inactive for seven days or more during code review.
+- **How to sign up**: It not yet clarified how, maybe you can mail them the URLs of PR.
```
Comment (oleg-nenashev) [inline README.md]: https://jenkins.io/events/hacktoberfest/#quick-start

File: README.md [hunk 2]
```diff
@@ -149,6 +149,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+- **Issues**: [Good first issues on GitHub](https://github.com/search?q=org%3Ajenkinsci+org%3Ajenkins-infra+org%3Ajenkins-zh+is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
```
Comment (oleg-nenashev) [inline README.md]: https://jenkins.io/events/hacktoberfest/#where-to-contribute provides the actual list, GitHub issues is a subset

File: README.md [hunk 3]
```diff
@@ -149,6 +149,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [jabref](https://github.com/search?&q=user%3Ajabref+type:issue&state=open&type=Issues)
 - **Notes**: More details in [the blog post](https://www.jabref.org/hacktoberfest/2019.html).
 
+#### **Jenkins**
+
+- **Swag**: limited edition swag
```
Comment (oleg-nenashev) [inline README.md]: Stickers

File: README.md [hunk 4]
```diff
@@ -149,6 +149,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
...
+- **Requirements**: PR to any repositories under [any repo of Jenkins](https://issues.jenkins-ci.org/browse/JENKINS-59817?filter=-4). The PR must be at least raised in October 2019 and you should not be inactive for seven days or more during code review.
```
Comment (oleg-nenashev) [inline README.md]: Filter is incorrect, and  we do not guarantee swag for all contributors

File: README.md (hunk 3 above)
Comment (oleg-nenashev) [reply README.md]: T-shorts as well actually

File: README.md (hunk 4 above)
Comment (oleg-nenashev) [reply README.md]: So far I can guarantee swag for Top 20 contributors, working on the budget things to get more

**Comment by crweiner** [comment]
Thank you so very much @oleg-nenashev for not only reviewing this PR, but also for celebrating Hacktoberfest with us all and Jenkins! @ManthanKeim, please make the changes as noted so we can get this merged and let me know when it's ready for another look. :+1:

**Comment by ManthanKeim** [comment]
Thank you very much @oleg-nenashev sir and @crweiner for your comments, please let me know if I have made the suitable points in the latest commit.

**Comment by crweiner** [comment]
> Thank you very much @oleg-nenashev sir and @crweiner for your comments, please let me know if I have made the suitable points in the latest commit.

Hi @ManthanKeim! Can you please also add Jenkins to the **Least Involvement to Most Involvement** section of the list and double-check that you've changed everything @oleg-nenashev pointed out? Thank you!

**Comment by ManthanKeim** [comment]
> > Thank you very much @oleg-nenashev sir and @crweiner for your comments, please let me know if I have made the suitable points in the latest commit.
> 
> Hi @ManthanKeim! Can you please also add Jenkins to the **Least Involvement to Most Involvement** section of the list and double-check that you've changed everything @oleg-nenashev pointed out? Thank you!

@crweiner please check now, I hope I made the required changes.

**Comment by crweiner** [comment]
Thank you again @ManthanKeim and @oleg-nenashev!! I'm happy to get this live! Have a wonderful Hacktoberfest!! :+1: :1st_place_medal:

**Pull request merged**